pkg/filters/int.go

```diff
@@ -104,15 +103,17 @@ func (f *IntFilter) addNotEqual(val int64) {
 	f.notEqual[val] = true
 }
 
-func (f *IntFilter) addLesser(val int64) {
-	if val > f.lessThan {
-		f.lessThan = val
+func (f *IntFilter) addLesserThan(val int64) {
```

Comment:
You can replace Less->Lower in this commit as well

Reply:
Critical? I'd rather not deal with cropping up conflict errors in the rebase again, that's why I did the styling changes in one final commit.

Reply:
But this is the last commit - you can just ammend it without rebasing

Reply:
Oh you mean change LesserThan to LowerThan? I'd rather wait on the types change for that.